client/src/components/DisplayBookings/CardImage/CardImage.tsx

```diff
@@ -0,0 +1,17 @@
+import clsx from 'clsx';
+import useStyles from './useStyle';
+
+interface Props {
+  isUpcoming?: boolean;
+}
+
+export default function CardImage({ isUpcoming }: Props): JSX.Element {
```

Comment:
We don't need a component here as we are only returning an HTML image element :)

Reply:
You can use the HTML image element directly within the component you intend to use it :)